### PR TITLE
[map] url param to set the stat var for installations on a map

### PR DESCRIPTION
**Pull request by chejennifer**
- add a url param to set the stat var used for the installations on a map. If no stat var set, will default to using the same stat var as being shown for the map regions.
- URL params to show glaciers: &mapsv=SurfaceArea&ppt=Glacier
- Now that installations can show a different stat var than the rest of the map, remove the dateParam when retrieving installations data because it can be hard to make the date work for both the map stat var and the installations stat var. For example, RCP stat vars use 2050 as the date, but not many other stat vars have data for 2050.
Screen Shot 2022-01-24 at 1 02 31 PM

### CODE REVIEW

File: static/js/tools/map/chart_loader.tsx [hunk 1]
```diff
@@ -339,10 +340,11 @@ function fetchData(
       return resp.data;
     });
 
+  const mapPointSv = statVar.mapPointSv || statVar.dcid;
   const mapPointDataPromise: Promise<GetStatSetResponse> = placeInfo.mapPointPlaceType
     ? axios
         .get(
-          `/api/stats/within-place?parent_place=${placeInfo.enclosingPlace.dcid}&child_type=${placeInfo.mapPointPlaceType}&stat_vars=${statVar.dcid}${dateParam}`
+          `/api/stats/within-place?parent_place=${placeInfo.enclosingPlace.dcid}&child_type=${placeInfo.mapPointPlaceType}&stat_vars=${mapPointSv}`
```
Comment (beets) [inline static/js/tools/map/chart_loader.tsx:347]: should we keep the dateParam?

**Review by beets — APPROVED** [review]
nice!

File: static/js/tools/map/chart_loader.tsx (hunk 1 above)
Comment (chejennifer) [reply static/js/tools/map/chart_loader.tsx:347]: keeping dateParam doesn't work well when the stat var for the map is different for the installations .. for example, RCP statvars will use 2050, but not many other stat vars will have data for 2050

Comment (beets) [reply static/js/tools/map/chart_loader.tsx:347]: that makes sense. could you update the pr description with the update? it's good to list all these changes out, which helps with future debugging, and/or a comment in code.

Comment (chejennifer) [reply static/js/tools/map/chart_loader.tsx:347]: done, updated pr description

**Pull request merged**